earth_enterprise/src/portableserver/README.md

```diff
@@ -64,15 +64,17 @@ If you're running an old version of a Red Hat distribution, such as Cent OS or R
 
 Run
 
-    earthenterprise/earth_enterprise/src/portableserver/build.py
+    cd earthenterprise/earth_enterprise/src/portableserver
```

Comment:
Shouldn't this change also be done for the Windows build?

https://github.com/google/earthenterprise/pull/1791/files#diff-7b246b4ec3f731576c092fc92236f59fR143